### PR TITLE
Adding unit tests for the main scripts (ccwrite and ccget)

### PR DESCRIPTION
**Pull request by pasenor**
Adding unit tests for ccget and ccwrite, using the first approach outlined in #472 

I moved the scripts to `*py` files to enable a cleaner importing, since the `import` statement does not recognise files without extension and we would have to do something like

    imp.load_source('scripts/ccwrite)

which is arguably less clean and deprecated in python 3

### CODE REVIEW

File: test/io/testscripts.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+"""Unit tests for main scripts (ccget, ccwrite)."""
+import os
+import unittest
+from io import StringIO
+
+try:
+    from unittest.mock import patch
+except ImportError:
+    from mock import patch
+
```
Comment (langner) [inline test/io/testscripts.py]: You don't need that many blank lines between sections. Please follow the recommendations of PEP8 for formatting.

**Comment by langner** [comment]
Travis is failing, so I presume this needs to wait for some other fixes?

**Comment by pasenor** [comment]
> Travis is failing, so I presume this needs to wait for some other fixes?

This PR is tests only, not including the fixes to ccwrite from #466, so it fails predictably with

    Traceback (most recent call last):
      File "test/io/testscripts.py", line 36, in setUp
        from scripts import ccget
      File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/cclib-1.5.2-py2.7.egg/scripts/__init__.py", line 9, in <module>
        from . import ccwrite
      File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/cclib-1.5.2-py2.7.egg/scripts/ccwrite.py", line 45
        type=int,
           ^
    SyntaxError: invalid syntax

**Comment by ATenderholt** [comment]
It's also failing at another spot, which I think hints at another problem related to the changes in this PR.

```
Traceback (most recent call last):
  File "test/io/testscripts.py", line 50, in setUp
    from scripts import ccwrite
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/cclib-1.5.2-py2.7.egg/scripts/__init__.py", line 8, in <module>
    from . import ccget
ImportError: cannot import name ccget
```

File: test/io/testscripts.py (hunk 1 above)
Comment (pasenor) [reply test/io/testscripts.py]: fixed

**Comment by pasenor** [comment]
> It's also failing at another spot, which I think hints at another problem related to the changes in this PR.

This is another facet of the same problem: due to syntax errors in the script file it cannot be imported. Now that the script is fixed it goes away, although I agree it was not the most sensible message to fail with. Added checks around import statements.

**Comment by berquist** [comment]
I am ok with this. Is everyone else?

**Comment by ATenderholt** [comment]
Looks good to me.

**Pull request merged**